### PR TITLE
make sure boolean attributes are saved in a standards compatible way

### PR DESCRIPTION
**Pull request by triglian**
PTAL @youyix

### CODE REVIEW

File: lib/asqExercisePlugin.js [hunk 1]
```diff
@@ -219,11 +232,56 @@ module.exports = ASQPlugin.extend({
...
+    var attr = this._assureBooleanValFromHTMLBooleanAttr($el.attr())
 
     return this.dashed2Camel(attr);
   },
 
+  /*
+  * boolean attributes in HTML can have no value,
+  * or have as value the attribute names. Cheerio
+  * messes up the first category, reporting them 
+  * as haveing an empty string for value.
```
Comment (youyix) [inline lib/asqExercisePlugin.js]: small typo: having ;-)

**Comment by youyix** [comment]
It looks good but just one suggestion: the methods for working with cheerio are possible to be used in other elements. We need find a way to avoid duplicated code. (Also the same for the some other methods like `camel2Dashed`)

File: lib/asqExercisePlugin.js (hunk 1 above)
Comment (triglian) [inline lib/asqExercisePlugin.js]: fixed

**Comment by triglian** [comment]
I agree, we can create a small utility lib for working with serverside ASQ plugins

**Pull request merged**